raya/temperature.py

```diff
@@ -0,0 +1,15 @@
+import numbers 
+
+celsius_100 = print(100 * 5 / 9 - 32)
+# A float is printed to the console and you are able to
+# notice because of the decimal followed by numerous integers
+celsius_0 =  print(0 * 5 / 9 - 32)
+print( 34.2 * 5 / 9 -32)
+convert_5C = print(5 * 9 / 5 + 32)
+# boolean1 = print(int(float(bool(30.2 * 9 / 5 + 32 == 85.1 * 5 /9 - 32))))
+
+temperature_1 = int(85.1)
+temperature_2 = int(30.2 * 9 / 5 +32)
+boolean_1 = print(bool(temperature_1 > temperature_2))
+# Temperature_1 is the Fahrenheit comparison to Temperature_2 "30.2 celsius" 
+# You will notice that 30.2 Celsius is warmer than 85.1
```

Comment:
good! But the formula was misapplied. Make sure to follow the correct formula/order of operations